### PR TITLE
Switch redis cache and brainzutils.cache (GCI17)

### PR DESCRIPTION
**Pull request by mikhailyumanov**
# Summary

* This is a…
    * [x] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

* JIRA ticket (_optional_): 

# Solution

# Action

### CODE REVIEW

File: listenbrainz/webserver/__init__.py [hunk 1]
```diff
@@ -101,7 +101,20 @@ def gen_app(config_path=None, debug=None):
 
     # Redis connection
     create_redis(app)
-
+    
+    # Cache 
+    if 'REDIS_HOST' in app.config and\
+       'REDIS_PORT' in app.config and\
+       'REDIS_NAMESPACE' in app.config and:
```
Comment (paramsingh) [inline listenbrainz/webserver/__init__.py]: Problem seems to be the extra and here

**Comment by naiveaiguy** [comment]
Do keep in mind [git commit message guidelines](https://chris.beams.io/posts/git-commit/) as well.

**Comment by mikhailyumanov** [comment]
Okay, I will keep it in mind, thank you

**Comment by paramsingh** [comment]
Closing this in favor of #333 for now.

**Pull request closed without merging**